src/view/pane/chat.js

```diff
@@ -550,10 +550,11 @@ Candy.View.Pane = (function(self, $) {
        * Show the login form modal
        *
        * Parameters:
-       *  (String) message - optional message to display above the form
+       *  (String) messageKey - optional message to display above the form
        *  (String) presetJid - optional user jid. if set, the user will only be prompted for password.
        */
-      showLoginForm: function(message, presetJid) {
+      showLoginForm: function(messageKey, presetJid) {
+        var message = $.i18n._(messageKey);
```

Comment:
Sounds like you have a good handle on it then.  I was referring to the first assignment (args = args || {}) as being necessary, if args is undefined args.presetJid will error.

Thanks for the professional response.  I haven't had my code called 'messy' in decades.  Frankly it hurt.  I tried to be sociable earlier (regarding tabs), he shot that down too.  I'm not getting paid, so it had better be pleasant; if neither applies, I have other things i can do.  But I do wish you well with the project.